source/sdk/math.ooc

```diff
@@ -157,7 +158,11 @@ extend Double {
 	clamp: func (floor, ceiling: This) -> This {
 		this > ceiling ? ceiling : (this < floor ? floor : this)
 	}
-	equals: func (other: This, tolerance := 0.0001) -> Bool { (this - other) abs() < tolerance }
+	equals: func (other: This, tolerance := This defaultTolerance) -> Bool { (this - other) abs() < tolerance }
```

Comment:
Sure absolute error can't cover all possible use cases, but I guess that's better than nothing